ctaplot/io/io.py

```diff
@@ -16,40 +16,53 @@ def read_lst_dl1_data(filename, key='dl1/event/telescope/parameters/LST_LSTCam')
     """
 
     data = pd.read_hdf(filename, key=key)
-    # data.rename({
-    #
-    # })
+
+    data = data.rename(columns={
+        "true_alt": "mc_alt",
+        "true_az": "mc_az",
+        "reco_alt": "reco_alt",
+        "reco_az": "reco_az",
+        "gammaness": "reco_gammaness",
+    })
+
+    if data['true_energy'].min() > 0.1 and data['true_energy'].max() < 10:
+        # energy is probably in log(GeV)
+        data['reco_energy'] = 10 ** (data['reco_energy'] - 3)
+        data['true_energy'] = 10**(data['true_energy'] - 3)
+
     return data
 
 
-def read_lst_dl2_data(filename, key='dl2/event/telescope/parameters/LST_LSTCam'):
+def read_glearn_dl2_data(filename):
```

Comment:
Can you add the GammaLearn version in the docstring please?

Reply:
The function reads both version of the files produced by gammalearn 0.4 and 0.5